assets/scss/style.scss

```diff
@@ -0,0 +1,4 @@
+@import 'maps';
+@import 'mixins';
+@import 'base';
+@import 'header';
```

Comment:
The files already have _ and they are compiled into a single css file. From what I read, the _ is not required at import. The underscore is only necessary at import when you have e.g. header.scss and _header.scss, otherwise, the following import will give exactly the same results:
```
@import 'header';
@import '_header';
```